packages/edit-site/src/components/sidebar-navigation-screen-global-styles/index.js

```diff
@@ -69,15 +77,67 @@ function SidebarNavigationScreenGlobalStylesContent() {
 	// rendering the iframe. Without this, the iframe previews will not render
 	// in mobile viewport sizes, where the editor canvas is hidden.
 	return (
-		<BlockEditorProvider
-			settings={ storedSettings }
-			onChange={ noop }
-			onInput={ noop }
-		>
-			<div className="edit-site-sidebar-navigation-screen-global-styles__content">
+		<>
```

Comment:
Oh, we don't! I originally had the extra components in this PR added in the `content` section, but have since moved them to the footer. I've removed the extra fragment in https://github.com/WordPress/gutenberg/pull/51149/commits/ae88d0ba5f9d71b436da3ebe778a97a77397d8e7